routes/profileRoutes.js

```diff
@@ -16,4 +19,17 @@ router.route('/profile')
     nameValidator,
     updateProfile
   );
+
+router.route('/profiles/:username/follow')
+  .all(verifyToken)
+  .post(canFollowUser, addFollower);
+
+router.route('/profile/following')
+  .all(verifyToken)
```

Comment:
Since this and the one beneath don't have extra requests on the same route, you can do without `.all` here. Just a `.get` which will contain the middleware and the controller.